crates/solvers/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 // TODO remove this once the crate stabilizes a bit.
 #![allow(dead_code)]
+#![recursion_limit = "256"]
```

Comment:
Why?

Reply:
Also curious about this one.

Reply:
Was required by some `json!` macro invocation for some reason. :shrug: